### PR TITLE
Eth call fix2

### PR DESCRIPTION
**Pull request by lupin012**
Modified json interface to be compliant with EIP : Eip2930, Eip1559

### CODE REVIEW

File: silkrpc/json/types.cpp [hunk 1]
```diff
@@ -150,22 +153,44 @@ void to_json(nlohmann::json& json, const Transaction& transaction) {
...
     json["s"] = silkrpc::to_quantity(silkworm::rlp::big_endian(transaction.s));
 }
 
+
+
 } // namespace silkworm
 
 namespace silkrpc {
 
+uint64_t get_block_sizes(const Block &b) {
```
Comment (canepat) [inline silkrpc/json/types.cpp:179]: I would rename it `get_block_size` and put it in `Block` struct

**Review by canepat — CHANGES_REQUESTED** [review]
Add missing unit tests to avoid coverage diff

File: silkrpc/json/types.cpp [hunk 2]
```diff
@@ -150,22 +153,44 @@ void to_json(nlohmann::json& json, const Transaction& transaction) {
         json["to"] =  nullptr;
     }
     json["type"] = silkrpc::to_quantity(transaction.type.value_or(0));
+
+    if (transaction.type && transaction.type == silkworm::kEip1559TransactionType) {
```
Comment (canepat) [inline silkrpc/json/types.cpp:157]: First leg of AND is redundant

Comment (lupin012) [reply silkrpc/json/types.cpp:157]: ok

File: silkrpc/json/types.cpp (hunk 1 above)
Comment (lupin012) [reply silkrpc/json/types.cpp:179]: renamed, but Block is struct of silkworm not silkrpc.

Comment (canepat) [reply silkrpc/json/types.cpp:179]: Don't think so as it's defined in `silkrpc` namespace...

Comment (lupin012) [reply silkrpc/json/types.cpp]: Sorry my mistake

**Review by canepat — APPROVED** [review]

**Pull request merged**